featuretools/tests/entityset_tests/test_dask_es.py

```diff
@@ -2,66 +2,73 @@
 import pandas as pd
 import pytest
 
+import woodwork.logical_types as ltypes
+
 import featuretools as ft
 from featuretools.entityset import EntitySet
 
 
-def test_create_entity_from_dask_df(pd_es):
+def test_add_dataframe(pd_es):
     dask_es = EntitySet(id="dask_es")
-    log_dask = dd.from_pandas(pd_es["log"].df, npartitions=2)
-    dask_es = dask_es.entity_from_dataframe(
-        entity_id="log_dask",
+    log_dask = dd.from_pandas(pd_es["log"], npartitions=2)
+    semantic_tags = {}
+    for col_name in pd_es["log"].columns:
+        semantic_tags[col_name] = pd_es["log"].ww.semantic_tags[col_name] - {'time_index', 'index'}
```

Comment:
Use the `get_df_tags` function here?

Reply:
good call, adding here and in the koalas tests